tmeister/static/src/toggles/table/toggle-table.helpers.spec.js

```diff
@@ -137,39 +137,46 @@ describe('toggle-table.helpers.js', () => {
 				},
 				{
 					toggle: {
-						env: "Production",
-						feature: "f3",
+						env: "Integration",
+						feature: "f2",
 						state: "OFF",
 					},
 				},
 				{
 					toggle: {
 						env: "Stage",
-						feature: "f4",
+						feature: "f2",
 						state: "OFF",
 					},
 				},
 				{
 					toggle: {
-						env: "Stage",
-						feature: "f5",
+						env: "Production",
+						feature: "f1",
 						state: "OFF",
 					},
 				},
 				{
 					toggle: {
 						env: "Stage",
-						feature: "f6",
+						feature: "f1",
+						state: "OFF",
+					},
+				},
+				{
+					toggle: {
+						env: "Integration",
+						feature: "f1",
 						state: "OFF",
 					},
 				},
-			);
+			];
 
-			envList = ["Production", "Stage"];
+			envList = ["Integration", "Stage", "Production"];
```

Comment:
The test verifies that the toggles come out in a reversed order: `Integration`, `Stage`, and `Production` instead of the order defined in the `toggles` array.